.env.example

```diff
@@ -0,0 +1 @@
+SECRET_API_KEY = "insert your secret API key here"
```

Comment:
Maybe don't add `"` in the example? I'm afraid it will be misleading

Reply:
so it should be only `SECRET_API_KEY=` ?

Reply:
or `SECRET_API_KEY=""`?